diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigJsonTest.kt

```diff
@@ -0,0 +1,42 @@
+package org.cqfn.diktat.ruleset.utils
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.common.config.rules.RulesConfigReader
+import org.cqfn.diktat.common.config.rules.getRuleConfig
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.junit.Test
+
+/**
+ * Special test that checks that developer has not forgotten to add his warning to a rules-config.json
+ * This file is needed to be in tact with latest changes in Warnings.kt
+ */
+class RulesConfigJsonTest {
+    @Test
+    fun `read rules config json`() {
+        val allRulesFromConfig = readAllRulesFromConfig()
+        val allRulesFromCode = readAllRulesFromCode()
+
+        allRulesFromCode.forEach { rule ->
```

Comment:
I believe we also need the opposite check - that no old warning IDs are left in config file.

Reply:
agree - will add

Reply:
Found 2 bugs in both config and docs, fixed!